apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/model/OpphoerBrevData.kt

```diff
@@ -36,12 +37,20 @@ data class AdopsjonRevurderingBrevdata(
             }
 
             return AdopsjonRevurderingBrevdata(
+                innhold = listOf(),
                 virkningsdato = behandling.virkningsdato!!.atDay(1),
                 adoptertAv1 = behandling.revurderingInfo.adoptertAv1,
                 adoptertAv2 = behandling.revurderingInfo.adoptertAv2
             )
         }
     }
+
+    override fun flett(text: String?) = when (text) {
+        "<dato>" -> SlateFletter.formaterDato(virkningsdato, Spraak.NB)
+        "<adopsjonsdato>" -> "<her skal adopsjonsdato komme automatisk>"
+        "<navn>" -> SlateFletter.formaterNavn(adoptertAv1, adoptertAv2)
```

Comment:
Ville kanskje spesifisert placeholder-nøklene ytterligere. F.eks. `<virkningsdato>` og `<adoptivforeldre_navn>`, siden navn og dato er litt vel generisk og kan være hva som helst.

Reply:
Kanskje, men i konteksten dei her er spesifisert (i koden for akkurat dette brevet) er dei unike, og det matchar med placeholdarane Merethe og Liv Inger har brukt i sjølve malen: https://navno.sharepoint.com/:w:/r/sites/Pensjonsomrdet/_layouts/15/Doc.aspx?sourcedoc=%7B50096120-14AB-40D8-A189-F668F1DB0A88%7D&file=GBP%20-%20Opph%C3%B8r%20-%20Adopsjon%20-%20NB.docx&action=default&mobileredirect=true

Eg ser eigentleg for meg at vi bruker samme placeholders i json-representasjonane våre som dei har gjort i sin definisjon av malen

Reply:
Ser at 100% match ikkje funkar, for i omgjøring av farskap-malen brukast <navn> om både noverande og forrige far 🤔